src/js/edu-benefits/1990e/config/form.js

```diff
@@ -116,6 +128,72 @@ const formConfig = {
     educationHistory: {
       title: 'Education History',
       pages: {
+        educationHistory: {
+          path: 'education-history',
+          title: 'Education History',
+          initialData: {
+          },
+          uiSchema: {
+            highSchoolOrGedCompletionDate: date.uiSchema('When did you earn your high school diploma or equivalency certificate?'),
+            'view:educationHistory': {
+              'ui:title': 'Please list any post-high school trainings you have completed.'
+            },
+            postHighSchoolTrainings: {
+              'ui:title': 'Education after high school',
+              'ui:options': {
+                itemName: 'Training',
+                viewField: EducationView,
+                hideTitle: true,
+              },
+              items: {
+                name: {
+                  'ui:title': 'Name of college, university or other training provider'
+                },
+                city: {
+                  'ui:title': 'City'
+                },
+                state: {
+                  'ui:title': 'State'
+                },
+                dateRange: dateRange.uiSchema(
+                  'From',
+                  'To'
+                ),
+                hours: {
+                  'ui:title': 'Hours completed'
+                },
+                hoursType: {
+                  'ui:title': 'Type of hours',
+                  'ui:options': {
+                    labels: hoursTypeLabels
+                  }          
+                },
+                degreeReceived: {
+                  'ui:title': 'Degree, diploma or certificate received'
+                },
+                major: {
+                  'ui:title': 'Major or course of study (NOT for high school)'
```

Comment:
Some of these fields are conditional depending on if you're in high school on the 5490, not sure if it's the same here. @goldenmeanie